packages/vx-clip-path/package.json

```diff
@@ -89,5 +89,8 @@
       "raf/polyfill",
       "<rootDir>/test/enzyme-setup.js"
     ]
+  },
+  "dependencies": {
+    "prop-types": "^15.6.2"
```

Comment:
can remove this as it's already included https://github.com/hshoff/vx/pull/392/files#diff-8b568cc49c33b4133a4de279448cea60R38